freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UsersyncPoller.java

```diff
@@ -45,31 +50,28 @@
     @Inject
     private UmsEventGenerationIdsProvider umsEventGenerationIdsProvider;
 
-    @Value("${freeipa.syncoperation.poller.enabled:true}")
-    private boolean enabled;
-
     @Inject
     private EntitlementService entitlementService;
 
-    @Scheduled(fixedDelayString = "${freeipa.syncoperation.poller.fixed-delay-millis:60000}",
-            initialDelayString = "${freeipa.syncoperation.poller.initial-delay-millis:300000}")
+    @Scheduled(fixedDelayString = "${freeipa.usersync.poller.fixed-delay-millis:60000}",
```

Comment:
can we remove the default value from here if it's defined in the `application.yml`?

Reply:
done